tests/packageurls

```diff
@@ -1767,6 +1767,7 @@ http://pypi.debian.net/vine/vine-1.1.4.tar.gz,vine,1.1.4
 http://pypi.debian.net/virtualenv/virtualenv-15.1.0.tar.gz,virtualenv,15.1.0
 https://virt-manager.org/download/sources/virt-viewer/virt-viewer-6.0.tar.gz,virt-viewer,6.0
 https://pypi.debian.net/voluptuous/voluptuous-0.10.5.tar.gz,voluptuous,0.10.5
+https://github.com/felixonmars/volume_key/archive/volume_key-0.3.9.tar.gz,volume_key,0.3.9
```

Comment:
nitpick: this should go one line up since volume_key comes before voluptuous alphabetically

Reply:
sure